korge-core/src/korlibs/image/bitmap/BitmapExt.kt

```diff
@@ -53,8 +53,8 @@ fun Bitmap.putSliceWithBorder(x: Int, y: Int, bmp: BmpSlice, border: Int = 1) {
     // Vertical replicate
     for (n in 1..border) {
         val rwidth = width + border * 2
-        this.copy(x, y, this, x, y - n, rwidth, 1)
-        this.copy(x, y + height - 1, this, x, y + height - 1 + n, rwidth, 1)
+        this.copy(x - border, y, this, x - border, y - n, rwidth, 1)
```

Comment:
Shouldn't be `rwidth + border * 2`?

Reply:
What do you exatly mean should I put that term?
I did the fix in a rush. Let me check it carefully with a bit more time on Friday :)
I will create also a "better" (more colorful) tile for visually checking if the expansion of the border works as we expect.

Reply:
```
fun Bitmap.copy(srcX: Int, srcY: Int, dst: Bitmap, dstX: Int, dstY: Int, width: Int, height: Int)
```

I mean, if border was not taken into account and if position is displaced now, width might need to be extended?

For example, let's imagine a segment: x=10, width=10 (right = 20)
If I want to add one pixel on both sides to the line: x=9, width=12 (right=21)

So subtracting `N` to the `x` (`left`), requires adding `N*2` to the width, or N to the `right` coordinate.
In the case we are extending the same amount of pixels on both sides of the segment.

Not sure if that was the case here, but asked just in case.

Reply:
You are right. I checked now after my Friday stream again and my implementation actually adds the border twice. So, if I add border = 2 than 4 pixels are added to each side of the tile :)

I need to check again...

Reply:
I checked again now and I could not find a problem anymore with my initial changes. Here is a screenshot for border = 1:

![image](https://github.com/korlibs/korge/assets/19785115/0bbdc011-6340-4e51-be05-1a58af16c783)

So it expands correctly by one pixel in each direction. I checked that visually with border = 2, 3 and 5 and it was correctly expanding with 2 resp. 3 and 5 pixels.